docs/guide/setup/creating_ligand_networks.rst

```diff
@@ -1,6 +1,51 @@
 .. _define_ligand_network:
+.. _userguide_ligand_network:
 
 Defining the Ligand Network
 ===========================
+A ligand network is a set of small molecules connected by mappings of two ligands.
+Such a network can represent a set of drug candidates derived from molecule enumeration that should
```

Comment:
This sentence is very long / difficult to read, I would suggest splitting up or rewording.

Reply:
tried to improve ti